mysql-test/suite/s3/suite.pm

```diff
@@ -4,5 +4,45 @@ package My::Suite::S3;
 
 return "Need S3 engine" unless $::mysqld_variables{'s3'} eq "ON" or $ENV{HA_S3_SO};
 
+if (!$ENV{'S3_HOST_NAME'})
+{
+  return "Environment variable S3_HOST_NAME need to be set";
+}
+
+if (!$ENV{'S3_BUCKET'})
+{
+  return "Environment variable S3_BUCKET need to be set";
+}
+
+if (!$ENV{'S3_REGION'})
+{
+  return "Environment variable S3_REGION need to be set";
```

Comment:
https://mariadb.com/kb/en/s3-storage-engine-system-variables/#s3_region this defaults to blank. At least the MinIO can handle a blank region.  Can Huawei handle a blank region?

So just set an empty env variable here.